src/gpu/generic/sycl/ref_convolution.hpp

```diff
@@ -110,7 +110,10 @@ struct ref_convolution_fwd_t : public gpu::generic::sycl::primitive_t {
                     VERBOSE_UNSUPPORTED_ATTR);
             VDISPATCH_CONV(
                     IMPLICATION(!attr()->scales_.has_default_values(),
-                            attr_scales_ok()
+                            attr_scales_ok({{DNNL_ARG_SRC, {0}},
+                                    {DNNL_ARG_WEIGHTS, {0}},
+                                    {DNNL_ARG_DST, {0}}})
+                                            == status::success
```

Comment:
Sorry, my bad again. I asked if we can add the build for generic backend in PreCI to have less escapes like this.

It can be replaced as this:
```
CHECK(attr_scales_ok({{DNNL_ARG_SRC, {0}}, {DNNL_ARG_WEIGHTS, {0}}, {DNNL_ARG_DST, {0}}}));
```
Verbose checks will be inside the call, no need to have an extra one outside of it.

Reply:
Change made as suggested, thanks.